modules/apigee_edge_teams/tests/src/Functional/UiTest.php

```diff
@@ -192,17 +192,11 @@ protected function teamsWorkflowTest() {
     $this->assertSession()->pageTextContains($this->fields['email']['data_edited']);
 
     // Add the other user as a member to the team.
-    $this->clickLink('Members');
-    $this->assertSession()->pageTextContains($this->account->getAccountName());
-    $this->clickLink('Add members');
-    $anotherEmail = $this->randomMachineName(10) . '@example.com';
-    $this->submitForm([
```

Comment:
I removed the add members form test since the UI has changed. We are going to test this separately.